cmake/kokkos_enable_options.cmake

```
@@ -104,7 +104,21 @@ kokkos_enable_option(
   HIP_MULTIPLE_KERNEL_INSTANTIATIONS OFF
   "Whether multiple kernels are instantiated at compile time - improve performance but increase compile time"
 )
-kokkos_enable_option(IMPL_HIP_MALLOC_ASYNC ${KOKKOS_ENABLE_HIP} "Whether to enable hipMallocAsync")
+# FIXME_HIP
+if(KOKKOS_ENABLE_HIP)
+  #here just for the version, can be removed with the fixme as it will be found by our TPL processing
+  find_package(hip 6.2 REQUIRED)
```

I think we should not specify a version here to be independent of how hip implements find_package 
```suggestion
  find_package(hip REQUIRED)
```